### ISSUE TITLE
Script and messaging to recruit donors in the interview process: HfLA website

### ISSUE THREAD

**Issue by Basu3040** [hackforla/website#2309]
### Overview

We need script and the messaging, and tools to be able talk with donors.

### Action Items

- [x] Create a script
- [x]  Edit the [existing research plan](https://docs.google.com/document/d/160-4HZmpLhLpVDsPmPg9Ivy3CZ9hn17xxJtZHGm4rXU/edit) to make it more directly targeted toward corporate donor research
- [ ] Get added to 1 password
- [ ] Talk with research lead to get access to siteresearch@hackforla.org email
- [ ] Create a [Calendly](https://calendly.com/app/signup?utm_medium=cpc&utm_source=google&utm_campaign=2074601018) for siteresearch@hackforla.org and update [this messaging](https://docs.google.com/document/d/1JyzHMXfANykFkwLENiSQYb5yCV0LCIO0i3FA5odgFM0/edit) with current details.
- [x] Add calendly login info to 1password
- [ ] Review the messaging with Product
- [ ] Align script questions with research objectives using [this spreadsheet](https://docs.google.com/spreadsheets/d/1hPpLu_RmaT1gKUxWPtU-5V_ZrsBZsxZCNK-3Un40WnE/edit#gid=0)
- [ ] Review the script with Product
- [ ] Email the donors to setup interviews
- [ ] Release dependency on #2704 

### Resources/Instructions
[Messaging draft](https://docs.google.com/document/d/1JyzHMXfANykFkwLENiSQYb5yCV0LCIO0i3FA5odgFM0/edit)
[Existing research plan](https://docs.google.com/document/d/160-4HZmpLhLpVDsPmPg9Ivy3CZ9hn17xxJtZHGm4rXU/edit)
[Script Draft](https://docs.google.com/document/d/1-IVw_GgMWOrB4mpO1dplNY2kSHJLLywlupD-DU-1htU/edit)
[Calendly Link](https://calendly.com/siteresearch/45?month=2022-02)

**Comment by Basu3040** [comment]
Progress: The script has been created with Calendely invite.
Blockers: None
ETA: 10/4/2021

**Comment by Basu3040** [comment]
Done!!

**Comment by sacamp** [comment]
Waiting for introduction to corporate donor representatives.

**Comment by sacamp** [comment]
Waiting for introduction to corporate donor representatives.

**Comment by sacamp** [comment]
Waiting for introduction to corporate donor representatives.

**Comment by tamalatrinh** [comment]
**Progress:** Was assigned issue 2 days ago (Feb 9)
**Blockers:** Waiting for/to discuss next steps at this week's Sunday 2/13 meeting 

- intros/ where to find interviewees
- agreeing on interview questions

**Availability:** Next 2 weeks?  I created a calendly for 2/15 - 3/1 with openings Tues - Thurs. 
Here's the live link: https://calendly.com/siteresearch/45?month=2022-02
**ETA:** Not sure since everything is contingent on finding interviewees + scheduling etc.

**Comment by tamalatrinh** [comment]
The calendly in the messaging document needs to be changed to the current calendly

**Comment by tamalatrinh** [comment]
update messaging with time options

**Comment by sacamp** [comment]
Hi @tamalatrinh, please remove the "2 weeks inactive" label after updating your issue. Thanks!

**Comment by tamalatrinh** [comment]
- Messaging has been updated with time option (highlighted in yellow)
       -New link to calendly has been updated

- Script draft has been updated 
       -'passionate' changed to 'care deeply about'
- Demographic questions will only be shared with **individual** donors

Waiting for product to review:
1. [Messaging](https://docs.google.com/document/d/1JyzHMXfANykFkwLENiSQYb5yCV0LCIO0i3FA5odgFM0/edit)
2. [Script](https://docs.google.com/document/d/1-IVw_GgMWOrB4mpO1dplNY2kSHJLLywlupD-DU-1htU/edit)

**Next steps:**
Possibly test run interview script to see how long interview will take. Current estimate length is 45mins, but we think could be shorter. Shorter time could mean more people would be open to chatting.

**Comment by ExperimentsInHonesty** [comment]
@tamalatrinh
We would like to try something new.  Please put next to each question the number of each objective that it relates to.  

Example:
- Script Section: Motivations for Giving
- Script Question: What information do you find most useful when learning about an organization you’re interested in?
- 3. Objectives b,ii

How we would want you to annotate the Script Question in the script document: 

> What information do you find most useful when learning about an organization you’re interested in? (3,b,ii)

Please use [this spreadsheet](https://docs.google.com/spreadsheets/d/1hPpLu_RmaT1gKUxWPtU-5V_ZrsBZsxZCNK-3Un40WnE/edit#gid=0) to organize the questions:

**Comment by sacamp** [comment]
@tamalatrinh Please use [this spreadsheet](https://docs.google.com/spreadsheets/d/1hPpLu_RmaT1gKUxWPtU-5V_ZrsBZsxZCNK-3Un40WnE/edit#gid=0) to organize the questions

**Comment by dionw412** [comment]
Availability for this week: Monday, Wednesday, Thursday
My estimated ETA for completing this issue: March 21, 2022

**Comment by dionw412** [comment]
Progress: Revised research plan to target corporate donors
ETA: March 30, 2022

**Comment by dionw412** [comment]
1. Progress: I have completed revising the research plan to target corporate donors. Will try to meet with Bonnie to discuss research plan
2. Blockers: N/A
3. I will work on this issue for 3 hours
4. ETA: I expect to complete this issue on 5/25/22
5. Pictures: None

**Comment by dionw412** [comment]
1. Progress: Updated [Research Plan to target corporate donors](https://docs.google.com/document/d/160-4HZmpLhLpVDsPmPg9Ivy3CZ9hn17xxJtZHGm4rXU/edit). Would like Bonnie to review for any questions
2. Blockers: None
3. Availablity: 1.5 -2 hours/ day (Tues, Wed, Thurs)
4. ETA: The issue should be completed by 5/25/22

**Comment by dionw412** [comment]
Progress: I will ask the UX lead permission to be added to 1 password and access to email.
Blockers: 
Availability: 2 hours on Wednesday, 2 hours on Friday
ETA: July 6, 2022
Pictures: None

**Comment by HackforLABot** [comment]
Hi @DakuwoN, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by DakuwoN** [comment]
I am available everyday after 3PM Central.
